src/Machine.js

```diff
@@ -652,6 +665,96 @@ class Machine {
     }
   }
 
+  /**
+   * Read dependencies JSON file content
+   * @param
+   * @return
+   * @private
+   */
+  _loadDependencies() {
+    if (!this.dependenciesUseFile) {
+      if (this.dependenciesSaveFile) {
+        this.dependencies = new Map();
+      }
+
+      return;
+    }
+
+    if (!fs.existsSync(this.dependenciesUseFile)) {
+      throw new Error(`The dependencies JSON file '${this.dependenciesUseFile}' does not exist`)
+    }
+
+    try {
+      this.dependencies = new Map(JSON.parse(fs.readFileSync(this.dependenciesUseFile, 'utf8')));
+    } catch(err) {
+      throw new Error(`The dependencies JSON file '${this.dependenciesUseFile}' cannot be used: ${err.message}`);
+    }
+  }
+
+  /**
+   * Read directives JSON file content and merge it with actual context
+   * @param {*} context
+   * @return {*}
+   * @private
+   */
+  _loadAndMergeDirectives(context) {
+    if (!this.directivesUseFile) {
+      if (this.directivesSaveFile) {
+        this.directives = { ...context };
```

Comment:
This line is causing automated tests to fail.

Reply:
Here is PR, which should fix an issue above:
https://github.com/electricimp/Builder/pull/62